src/migrations/migration_helpers.js

```diff
@@ -45,4 +51,4 @@ const deduplicateDocuments = async (db, collectionName, uniqueKey) => {
   }
 };
 
-export {collectionExists, safeDropIndex, deduplicateDocuments, safeCreateUniqueIndex};
```

Comment:
We can stop exporting `safeDropIndex`, as it is used only in migration_helpers now.